CMakeLists.txt

```diff
@@ -500,31 +479,6 @@ if(WIN32)
    set_property(TARGET fc PROPERTY SHARED_LIBRARIES_DEBUG ${SHARED_LIBRARIES_DEBUG})
    set_property(TARGET fc PROPERTY SHARED_LIBRARIES_RELEASE ${SHARED_LIBRARIES_RELEASE})
 
-endif(WIN32)
-
-SET(OPENSSL_CONF_TARGET )
```

Comment:
IIRC this solved the problem that on windows you couldn't use cli_wallet to connect to a `wss:` server. How does your change solve this?

Reply:
I did not check for this. It did not occur to me that this could cause such a problem.
 Do you have some `wss:` endpoints that I should definitely try ?

Reply:
I have an update.

I have tried both, the official release 3.0.0 `cli_wallet` binary and mine ( this branch, develop without the OpenSSL config ).
They seem to be working the same.

Tried to connect to a `wss:` endpoint ( `-s wss://bitshares.openledger.info/ws` ) and both binaries failed to connect.

This is beacause of the missing CA bundle. To correct for this, I have pointed to my bundle by setting the environment variable.

`set SSL_CERT_FILE=c:/ape/cacert.pem`

Then the both binaries worked. 

I have tried executing some simple commands, like `get_object 1.2.0` and both binaries succeeded.

This would confirm, I suppose, that the OpenSSL conf file is not the source of the problems.
Or is there some other case, which this could fail ?

Another improvement that comes to my mind is, that the binary could have used the system's certificate store.
But this is not in the scope of this pull request.

Reply:
Here's an explanation: https://github.com/bitshares/bitshares-core/issues/1147#issuecomment-406620270
@jmjatlanta are you using this in your windows builds?
If we remove it we must adapt the documentation.

Reply:
> IIRC this solved the problem that on windows you couldn't use cli_wallet to connect to a `wss:` server.

This (at least only this) didn't solve the `wss:` problem.

@cwyyprog can you help here? Why we needed the step copying openssl.cnf?

Reply:
TL;DR: I think we can accept this change.

The copying of `openssl.cnf` to binary directory was originally introduced in an old commit: https://github.com/bitshares/bitshares-fc/commit/04f07aa429c8f20d9afda4f3dda6fc01d8841715, to give the program which uses FC a way to specify a `openssl.cnf` file for OpenSSL initialization, probably because old windows machines weren't shipped with the file by default, and probably old version of openSSL lib required the file for initialization, or maybe the program need to specify something in the file. To specify the file, the program need to call `store_configuration_path()` which will set the `OPENSSL_CONF` environment variable which will be used by openssl lib (https://github.com/bitshares/bitshares-fc/commit/57e1d60257de85d2813fc24448b6fb4ab0d61f35).

In the beginning the copy was for all platforms. Later (still in old times) it was changed to Windows only in this commit: https://github.com/bitshares/bitshares-fc/commit/73dd2d68ea9a9297359afdffce58b7370663b7e2.

I don't have a clean windows machine right now, so can't check whether newer windows distributions contain an `openssl.cnf` file.

Anyway, we don't call `store_configuration_path()` in bitshares-core, which means the copied file wouldn't be used, and actually the Windows binaries still work, so it's no change if we don't copy that file at all.

Whether/how we can fix the `wss` issue is another story, whether we need the `openssl.cnf` file or need to modify it for the fix, is not in the scope of this pull request.